superchic.sh

```diff
@@ -1,6 +1,7 @@
 package: SuperCHIC
 version: "%(tag_basename)s%(defaults_upper)s"
-tag: "v2.04"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
```

Comment:
Or, alipwgmm (instead of alisw), whose field of competence is precisely generators (@qgp).